assets/js/components/setup/compatibility-checks.js

```diff
@@ -174,29 +174,32 @@ export default class CompatibilityChecks extends Component {
 				return <Fragment>
 					<p
 						dangerouslySetInnerHTML={ sanitizeHTML(
+							__( 'Looks like your site is having a technical issue with requesting data from Google services.', 'google-site-kit' ) +
+							'<br/>' +
 							sprintf(
 								/* translators: %1$s: Support Forum URL, %2$s: Error message */
-								__( 'Looks like your site is having a technical issue with requesting data from Google services.<br/>To get more help, ask a question on our <a href="%1$s">support forum</a> and include the text of the original error message:<br/>%2$s', 'google-site-kit' ),
+								__( 'To get more help, ask a question on our <a href="%1$s">support forum</a> and include the text of the original error message: %2$s', 'google-site-kit' ),
 								'https://wordpress.org/support/plugin/google-site-kit/',
-								error
+								'<br/>' + error
```

Comment:
Perhaps the translators comment should be updated to reflect that the placeholder value contains a new line?

Stylistically, perhaps we should use a template string here rather than `+` ? (just a thought)

E.g.
```js
dangerouslySetInnerHTML={ sanitizeHTML(
	`
	${ __( 'Looks like your site is having a technical issue with requesting data from Google services.', 'google-site-kit' ) }
	<br/>
	${ sprintf(
		/* translators: %1$s: Support Forum URL, %2$s: Error message */
		__( 'To get more help, ask a question on our <a href="%1$s">support forum</a> and include the text of the original error message: %2$s', 'google-site-kit' ),
		'https://wordpress.org/support/plugin/google-site-kit/',
		`<br/>${ error }`
	) }
	`,
	{
		ALLOWED_TAGS: [ 'a', 'br' ],
		ALLOWED_ATTR: [ 'href' ],
	}
) }
```

Reply:
I don't think that's necessary here, the translators shouldn't need to care about that.